charts/multicluster-scheduler/templates/cm.yaml

```diff
@@ -4,31 +4,5 @@ metadata:
   name: {{ include "fullname" . }}
   labels: {{ include "labels" . | nindent 4 }}
 data:
-  proxy-scheduler-config: |
-    apiVersion: kubescheduler.config.k8s.io/v1beta3
-    kind: KubeSchedulerConfiguration
-    leaderElection:
-      leaderElect: true
-      resourceName: admiralty-proxy-scheduler
-      resourceNamespace: {{ .Release.Namespace }}
-      resourceLock: leases
-    profiles:
-      - schedulerName: admiralty-proxy
-        plugins:
-          multiPoint:
-            enabled:
-              - name: proxy
-  candidate-scheduler-config: |
-    apiVersion: kubescheduler.config.k8s.io/v1beta3
-    kind: KubeSchedulerConfiguration
-    leaderElection:
-      leaderElect: true
-      resourceName: admiralty-candidate-scheduler
-      resourceNamespace: {{ .Release.Namespace }}
-      resourceLock: leases
-    profiles:
-      - schedulerName: admiralty-candidate
-        plugins:
-          multiPoint:
-            enabled:
-              - name: candidate
+  proxy-scheduler-config: {{ tpl (toYaml .Values.scheduler.proxySchedulerConfigYaml) . | indent 1 }}
+  candidate-scheduler-config: {{ tpl (toYaml .Values.scheduler.candidateSchedulerConfigYaml ) . | indent 1 }}
```

Comment:
nit (style) - this would read better with pipes all the way

```
toYaml .Values.scheduler.candidateSchedulerConfigYaml | tpl . | indent 2
```

also indent 2 would be more consistent with the current indentation